salt/modules/napalm_network.py

```diff
@@ -2018,6 +2033,13 @@ def load_template(template_name,
                 }
             )
     else:
+        salt.utils.versions.warn_until(
```

Comment:
I should have removed this in `Fluorine` (https://docs.saltstack.com/en/develop/topics/releases/fluorine.html#module-deprecations), but it turns out that it has more implications than I initially thoughts, postponing this till `Sodium` as well. Is that alright?

Reply:
:+1: